### PR TITLE
Add link to bulk downloads

### LINKED ISSUE #703: georgia: Add links to bulk downloads in docstrings
https://odapi.spa.ge/publication-policy (there's an SSL warning, but in Chrome I can click through to ignore it) mentions JSON and Excel bulk downloads.

Instead of adding a spider (they are not updating their data), I think it's fine to just mention in a docstring for now.

### PR DESCRIPTION
**Pull request by cecicasco**
closes #703

### CODE REVIEW

File: kingfisher_scrapy/spiders/georgia_records.py [hunk 1]
```diff
@@ -8,6 +8,11 @@ class GeorgiaRecords(LinksSpider):
     """
     Domain
       State Procurement Agency (SPA)
+    Bulk download documentation
```
Comment (yolile) [inline kingfisher_scrapy/spiders/georgia_records.py:11]: @jpmckinney just checking if this is what you wanted or https://odapi.spa.ge/publication-policy directly

Comment (jpmckinney) [reply kingfisher_scrapy/spiders/georgia_records.py:11]: https://odapi.spa.ge/spage.json.zip is a link to a bulk download, not to "bulk download documentation". So,  https://odapi.spa.ge/publication-policy is more appropriate.

File: docs/spiders.rst [hunk 2]
```diff
@@ -967,6 +967,9 @@ Tanzania
 
    scrapy crawl tanzania_zabuni
 
+Test
```
Comment (yolile) [inline docs/spiders.rst:970]: ?

File: kingfisher_scrapy/spiders/georgia_records.py (hunk 1 above)
Comment (yolile) [reply kingfisher_scrapy/spiders/georgia_records.py:11]: @cecicasco please change the docs to https://odapi.spa.ge/publication-policy

**Review by yolile — CHANGES_REQUESTED** [review]

**Comment by cecicasco** [comment]
perfect

**Review by yolile — APPROVED** [review]

**Comment by yolile** [comment]
@cecicasco actually, could you add the same documentation to `georgia_releases` too?

**Review by yolile — APPROVED** [review]

**Pull request merged**